### ISSUE TITLE
Problems running sample code with 

### ISSUE THREAD

**Issue by didacrios** [Adyen/adyen-secured-fields-sample-code#5]
Hello

With cloned version up to date trying to test index_with3DS2.php the demos does not work throwing the console the following Error

Our Merchant Account Name and also Api Key is correctly set in the `config/authentication.ini`

`adyen.js:formatted:7115 ERROR: SecuredFields configuration object does not have a configObject property`

**Comment by sponglord** [comment]
Hi,
Does the regular `index.php` page work?

**Comment by sponglord** [comment]
Actually, what I think this is - is an error with the `originKey`

Can you look in the Console at the Network tab and see whether the response from the call to `api/originKeys.php` contains any values?

Also if you're not testing from an `https://` environment you will have to edit `config.server.php` to generate an `http` value i.e. the `url` function should be like so:
```php
function url(){
    return sprintf(
        "%s://%s",
        'http', $_SERVER['HTTP_HOST']
    );
}
```

Of course, in the long run, this should always be running from an `https://` protocol ;-)

**Comment by didacrios** [comment]
It must be that, i'm trying in local and the server of the origin key is for another server :)

Thanks

**Comment by OZZlE** [comment]
This issue was a life saver to find on google but instead of closing it I would implement so if the API key is missing or wrong then you get a console.log 'Incorrect Adyen API credentials' (or similar) instead of this strange error... :)

**Comment by bazidev** [comment]
as @OZZlE said it was too hard to find the source of the bug, the message is guiding to other reasons, while the bug is due to api Key. i faced the same bug on SAP Hybris adyen plugin. and this thread saved me.

**Comment by sponglord** [comment]
When the next version of this sample is released (upgrading components to v3.6.1) it will also contain clearer messaging when something is not configured correctly in `authentication.ini` or `server.php`

**Comment by bazidev** [comment]
Hello @sponglord  i believe i've seen the same thing with adyen hybris plugin, will this fix it there too !?
